src/analysis/special_functions/arsinh.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2020 James Arthur. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: James Arthur, Chris Hughes, Shing Tak Lam.
+-/
+import analysis.special_functions.trigonometric
+import data.real.basic
+import analysis.special_functions.pow
+
+open real
+local attribute [pp_nodot] real.log
+noncomputable theory
+
+/-!
```

Comment:
The module doc usually goes before things like `open real` (but after the imports).

Reply:
I'm not going to push this because it's going to change directory, but thankyou!